lib/src/slices/api_knowledge/model/fingerprint/api_knowledge_model_fingerprint_rsp.dart

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) TIKI Inc.
+ * MIT license. See LICENSE file in root directory.
+ */
+
+import 'api_knowledge_model_fingerprint_rsp_edge.dart';
+
+class ApiKnowledgeModelFingerprintRsp {
+  Set<String>? fingerprints;
+  ApiKnowledgeModelFingerprintRspEdge? from;
+  ApiKnowledgeModelFingerprintRspEdge? to;
+
+  ApiKnowledgeModelFingerprintRsp({this.fingerprints, this.from, this.to});
+
+  ApiKnowledgeModelFingerprintRsp.fromJson(Map<String, dynamic>? json) {
```

Comment:
same here `ApiKnowledgeModelFingerprintRsp.fromJson(Map<String, dynamic> json)`, since it does nothing if json is null